src/components/DpDataTable/DpResizableColumn.vue

```diff
@@ -71,8 +78,12 @@ export default {
       this.cursorStart = e.pageX
       const resizeBound = this.resize.getBoundingClientRect()
       this.resizeWidth = resizeBound.width
-      const nextBound = this.nextEl.getBoundingClientRect()
-      this.nextWidth = nextBound.width
+
+      if (this.nextEl) {
```

Comment:
Why is it needed to reach out for the next resizable element here?

Reply:
if the current element's width has been changed, the width of the next element will be calculated and changed as well. I check if the `nextEl` exist, because it returns an `null` when the column is the last in the table. Otherwise, `getBoundingClientRect()` returns an issue.